src/main/java/it/unisa/Amigo/gruppo/controller/GruppoController.java

```diff
@@ -97,11 +138,127 @@ public String removeMembro(@PathVariable(name = "idPersona") int idPersona,@Path
+    public String closeCommissione(Model model, @PathVariable(name = "id2")int idCommissione){
+        Persona personaLoggata = gruppoService.getAuthenticatedUser();
+        prepareCandidateList(idCommissione,model,gruppoService.findAllMembriInSupergruppo(idCommissione));
+        model.addAttribute("isCapogruppo", gruppoService.isResponsabile(personaLoggata.getId(), gruppoService.findGruppoByCommissione(idCommissione).getId()));
+        model.addAttribute("isResponsabile", gruppoService.isResponsabile(personaLoggata.getId(),idCommissione));
+        model.addAttribute("flagChiusura",1);
+        gruppoService.closeCommissione(idCommissione);
+        return "gruppo/commissione_detail";
+    }
+
+    /**
+     * Gestisce il form necessario alla creazione di una nuova commissione @{@link Commissione}
+     * @param model per salvare informazioni da recuperare nell'html
+     * @param idSupergruppo l'id del gruppo al quale verrà assegnata la nuova commissione
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @GetMapping("/gruppi/{id}/commissioni/create")
+    public String createCommissioneForm(Model model, @PathVariable(name = "id")int idSupergruppo){
+        model.addAttribute("idGruppo", idSupergruppo);
+        model.addAttribute("command", new GruppoFormCommand());
+        return "gruppo/crea_commissione";
+    }
+
+    /**
+     * Gestisce la creazione di una commissione @{@link Commissione}
+     * @param gruppoFormCommand il gestore del form
+     * @param model per salvare informazioni da recuperare nell'html
+     * @param idGruppo l'id del gruppo al quale verrà assegnata la nuova commissione
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @PostMapping("/gruppi/{idGruppo}/commissioni/createCommissione")
+    public String createCommissione(@ModelAttribute("command") GruppoFormCommand gruppoFormCommand, Model model,@PathVariable(name = "idGruppo")int idGruppo ){
+        Commissione commissione = new Commissione(gruppoFormCommand.getName(), "Commissione", true, gruppoFormCommand.getDescrizione());
+        gruppoService.createCommissione(commissione, idGruppo);
+
+        model.addAttribute("idCommissione",commissione.getId());
+        model.addAttribute("idGruppo", idGruppo);
+
+        List<Persona> persone = gruppoService.findAllMembriInSupergruppo(idGruppo);
+        model.addAttribute("persone",persone);
+
+        return "gruppo/nomina_responsabile";
+    }
+
+    /**
+     * Nomina ed assegna un nuovo responsabile @{@link Persona}  ad una commissione @{@link Commissione}
+     * @param model per salvare informazioni da recuperare nell'html
+     * @param idCommissione la commissione che riceverà un nuovo responsabile
+     * @param idPersona id della persona che diventerà il nuovo responsabile
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @GetMapping("/gruppi/commissioni/{idCommissione}/nominaResponsabile/{idPersona}")
```

Comment:
Perché usare un metodo GET  e passare tutti sti parametri da url? Non è meglio creare un metodo mappato su POST che accetta la sottomissione da un form e creare un piccolo form in-line che sottomette il valore?